dev-db/sqlitebrowser/additional-files/sqlitebrowser.rdef

```diff
@@ -0,0 +1,64 @@
+
+resource app_flags B_SINGLE_LAUNCH | B_ARGV_ONLY;
+
+resource app_version {
+	major  = 3,
+	middle = 9,
+	minor  = 1,
+	
+	variety = B_APPV_ALPHA,
```

Comment:
I might be wroing, but I think that for the variety parameter we should use B_APPV_FINAL instead of B_APPV_ALPHA.